packages/flex-dev-utils/src/credentials.ts

```diff
@@ -52,12 +52,12 @@ const chooseAccount: Question = {
  * If no credentials exists, then prompts the user to enter the credentials
  */
 export const getCredential = async (): Promise<AuthConfig> => {
-  let accountSid;
-  let authToken;
+  let username;
+  let password;
 
-  const missingCredentials = !process.env.TWILIO_ACCOUNT_SID || !process.env.TWILIO_AUTH_TOKEN;
+  const missingCredentials = !((process.env.TWILIO_ACCOUNT_SID && process.env.TWILIO_AUTH_TOKEN) || (process.env.TWILIO_API_KEY && !process.env.TWILIO_API_SECRET));
```

Comment:
Let's add

```js
if (!process.env.TWILIO_USERNAME) {
  process.env.TWILIO_USERNAME = process.env.TWILIO_ACCOUNT_SID || process.env. TWILIO_API_KEY;
}
if (!process.env.TWILIO_PASSWORD) {
  process.env.TWILIO_PASSWORD = process.env. TWILIO_AUTH_TOKEN || process.env. TWILIO_API_SECRET;
}
```

Then we can just check `TWILIO_USERNAME/PASSWORD` thereafter and treat both as one.

Reply:
Actually, we are doing some cleanup in a followup PR; I'll do that then.